src/extension/content-script/batteries/helpers.ts

```diff
@@ -5,8 +5,8 @@ import type { Battery } from "~/types";
 import { ExtensionIcon } from "../../background-script/actions/setup/setIcon";
 
 export const findLightningAddressInText = (text: string): string | null => {
-  const match = text.match(/(⚡:?|lightning:|lnurl:)\s?(\S+@[\w-.]+)/i);
-  if (match) return match[2];
+  const match = text.match(/((⚡|⚡️):?|lightning:|lnurl:)\s?(\S+@[\w-.]+)/i);
```

Comment:
Surprisingly, these two are different characters!

Reply:
You have some context/docs to this?

Reply:
I think we ran into this problem before. some link with info to the character would be good.

Reply:
Nice, can you add a comment with the link to the regex?